### PR TITLE
chore(datagrid): update stories to CSFv3

### PR DESCRIPTION
**Pull request by spadidapelli**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-1525

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .storybook/stories/wizard/wizard-long-titles.stories.ts
Comment (williamernest) [inline .storybook/stories/wizard/wizard-long-titles.stories.ts:1]: Can you submit the wizard changes in it's own pull request?

Comment (spadidapelli) [reply .storybook/stories/wizard/wizard-long-titles.stories.ts:1]: Submitted new PR for wizard

**Review by mivaylo — APPROVED** [review]
